doc/developer-guides/hld/virtio-net.rst

```diff
@@ -509,6 +509,35 @@ is the virtual NIC created by acrn-dm:
       collisions:0 txqueuelen:1000
       RX bytes:0 (0.0 b) TX bytes:0 (0.0 b)
 
+How to Use MacVTap Interface
+============================
+In addition to Tap interface, ACRN also supports MacVTap interface. 
+MacVTap replaces the combination of the tap and bridge drivers with 
+a single module based on MacVLan driver. With MacVTap each
+virtual network interface is assigned its own MAC, IP address 
```

Comment:
delete the comma: ``MAC IP address``

Reply:
MAC and IP are different entities and so comma is needed here.

Reply:
Ah, so this is really two sentences then. OK.  Replace the comma with a period.